src/components/SelectionList/BaseSelectionList.js

```diff
@@ -250,7 +248,7 @@ function BaseSelectionList({
                 <CheckboxListItem
                     item={item}
                     isFocused={isFocused}
-                    onSelectRow={() => selectRow(item, index)}
+                    onSelectRow={() => selectRow(item)}
```

Comment:
I'm seeing that `CheckboxListItem` calls the `onSelectRow` passing the item, I think you can change to 
```
onSelectRow={selectRow}
```

Reply:
nice catch! removed the param and forgot to use the concise version - thanks!